cloud/ec2.go

```diff
@@ -1299,6 +1296,12 @@ func (m *ec2Manager) OnUp(ctx context.Context, h *host.Host) error {
 		"is_spot":       isHostSpot(h),
```

Comment:
should we move this logging into [the set-cloud-hosts-ready job](https://github.com/evergreen-ci/evergreen/blob/master/units/host_status.go#L171)? This would have the advantages that 1) it would be hit by Fleet as well and 2) this function would completely noop for anything but spot.

Reply:
I assume we don't really need this log. I don't think this log is really useful and there's already a [ticket](https://jira.mongodb.org/browse/EVG-13764) to make set-cloud-hosts-ready have more verbose logging so we can easily trace when a  host is up even without this log message.